app/views/reviews/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Add New Review</h2>
+<section>
+  <%= form_tag("/shelters/#{@shelter.id}/reviews", method: :post)  do %>
+
+    <%= label_tag "Title" %>
+    <%= text_field_tag :title %><br/>
+    <%= label_tag "Rating" %>
+    <%= text_field_tag :rating %><br/>
```

Comment:
buttons or drop down to limit ratings to 1-5

Reply:
One option is to add this as the first task in a refactor for styling phase. The use of a radio button is pretty straight forward since it is just html structuring that passes a string value anyway.

Reply:
I'd really like to use radio buttons here